### PR TITLE
[PyROOT] Resolve warnings about unused variables

### PR DESCRIPTION
**Pull request by hahnjo**

### CODE REVIEW

File: bindings/tpython/src/TPython.cxx [hunk 1]
```diff
@@ -352,6 +348,8 @@ void TPython::ExecScript(const char *name, int argc, const char **
    delete[] argv2;
 #else
 // TODO: fix this to work like above ...
+   (void)argc;
```
Comment (etejedor) [inline bindings/tpython/src/TPython.cxx:351]: `argc` is already used in code that will be compiled for both Py2 and Py3, so we don't need this here?

Comment (hahnjo) [reply bindings/tpython/src/TPython.cxx:351]: Yes and no: `argc` is "used but not set". This is the warning that I'm trying to address here.

Comment (etejedor) [reply bindings/tpython/src/TPython.cxx:351]: I see, so with `argc += 1` it's first used (read) to be set, but we still get the warning :)

**Review by etejedor — APPROVED** [review]

File: bindings/tpython/src/TPython.cxx (hunk 1 above)
Comment (hahnjo) [reply bindings/tpython/src/TPython.cxx:351]: Yes, recent Clang versions got a bit picky about those. I mean, it's correct in principle...

**Pull request merged**